model/aws/sdk-spans.yml

```diff
@@ -297,3 +297,57 @@ groups:
         requirement_level: recommended
       - ref: aws.s3.part_number
         requirement_level: recommended
+  
+  - id: span.aws.sqs.client
+    extends: span.aws.client
+    type: span
+    stability: development
+    span_kind: client
+    brief: "Attributes that exist for SQS request types."
+    attributes:
+      - ref: aws.sqs.queue.url
+        requirement_level: recommended
+      - ref: messaging.destination.name
+        requirement_level: recommended
+
+  - id: span.aws.sns.client
+    extends: span.aws.client
```

Comment:
I don't think the result looks right in the markdown - the span does not follow messaging semantic conventions. Take a look at other messaging spans - https://github.com/open-telemetry/semantic-conventions/tree/main/docs/messaging they inherit from one of the messaging groups and include typical messaging attributes. Here's e.g. [kafka yaml](https://github.com/open-telemetry/semantic-conventions/blob/78c42c576a25743902192466cf7ff81889bf3630/model/messaging/spans.yaml#L141)

Reply:
So for this, it's meant to extend the aws attributes like rpc system etc and not messaging attributes like message.operation. AFAIK, there isn't a way to extend multiple ones from different files no?

Reply:
there is no way to extend multiple, but you can extend one and reference attributes one-by-one (that are applicable to SNS/SQS), I would recommend extending messaging and this way avoiding referencing `rpc` attributes.

Reply:
I moved it to messaging to extend it there.